### ISSUE TITLE
Bar chart set label rotation and line break

### ISSUE THREAD

**Issue by shukerullah** [capitalone/react-native-pathjs-charts#142]
How to line break a bar chart label and remove rotation?

**Comment by marzolfb** [comment]
"Remove rotation" is a duplicate of #139 
What does "line break a bar chart label" mean - do you have an example? Is an example where you have some label consisting of multiple words and you want a line break in between the two words so you have a more compact display of the label? Or is it something else you are trying to do?

**Comment by shukerullah** [comment]
I have modified your code to use line break, see following image.

![graph](https://cloud.githubusercontent.com/assets/963765/25660591/667cfeca-3026-11e7-8409-faa41617676a.jpg)

**Comment by marzolfb** [comment]
What does "modified your code to use link break" mean? If you fixed something, would be willing to offer up your fix as a pull request?

**Comment by marzolfb** [comment]
@shukerullah With the latest commit that updated all the major dependencies used by this library, I suspect the line breaks won't appear like they do above with the latest version of this library unless you made an update to address that. Have you tried your example with the latest version of this library? Did you make a change to "fix" the display of line breaks in axis labels?

**Comment by shukerullah** [comment]
No, I haven't tried latest version, I will try and will send PR if line break works.
